components/typography/metadata/typography-body.yml

```diff
@@ -1,8 +1,9 @@
 name: Typography Body
-description: "Spectrum Typography Body."
-SpectrumSiteSlug: https://spectrum.corp.adobe.com/page/body/
+id: body-m
+description: Body is a typography component primarily used within Spectrum components and for blocks of text.
+SpectrumSiteSlug: https://spectrum.adobe.com/page/body/
 examples:
-  - id: body-1
```

Comment:
These were the old IDs, so the components were showing "Deprecated"

Reply:
ah ok, thanks for fixing.